src/main/webapp/css/user_style.css

```diff
@@ -0,0 +1,83 @@
+.main {
```

Comment:
Can you sort the CSS in this file in order of appearance in the HTML? It's taking me quite a while to find the rules I'm looking for.